lib/services/role_test.go

```diff
@@ -3035,9 +3049,14 @@ func BenchmarkCheckAccessToServer(b *testing.B) {
 	for n := 0; n < b.N; n++ {
 		for i := 0; i < 4000; i++ {
 			for login := range allowLogins {
-				if err := set.CheckAccessToServer(login, servers[i], AccessMFAParams{}); err != nil {
```

Comment:
This error check was added a few months ago in a linter warning clean up, and actually broke the benchmark. I went back to the commit that original added this benchmark and verified that it has been returning an error the entire time.